posts/2021-11-02-microprofile-jakarta-210012-beta.adoc

```diff
@@ -220,8 +220,8 @@ Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page]
 [#jakarta]
 == Jakarta EE 9 Beta Features package
 
-Open Liberty is the first vendor product to be Jakarta EE Web Profile 9.0 compatible since the 21.0.0.2-beta release. Open Liberty is also the first vendor product to be added to the link:https://jakarta.ee/compatibility/#tab-9[Jakarta EE Platform 9.0 compatability list], with the release of 21.0.0.3-beta. 
-Open Liberty 21.0.0.6-beta further expanded on this compatability by including new Jakarta EE9 supporting features, and 21.0.0.12-beta offers the same compatability with Jakarta EE9 with performance enhancements. 
+Open Liberty is the first vendor product to be Jakarta EE Web Profile 9.0 compatible since the 21.0.0.2-beta release. Open Liberty is also the first vendor product to be added to the link:https://jakarta.ee/compatibility/#tab-9[Jakarta EE Platform 9.0 compatability list], with the release of 21.0.0.3-beta.
```

Comment:
@mbroz2 Are you sure you don't want to link to https://jakarta.ee/compatibility/#tab-9_1

Reply:
I think both.  The previous link needs to remain as-is because that's what's being referenced (OL being first to get 9.0 certification).  However, we should also link to 9.1 in the blog.

Reply:
@jhanders34 was Open Liberty also first to be added to the Jakarta EE Platform 9.1 compatibility list, or just 9.0?

Reply:
I'm seeing "On day one of this new release, Open Liberty is one of the vendor implementations used to ratify the specification. " referring to 9.1

Reply:
```suggestion
Open Liberty is the first vendor product to be Jakarta EE Web Profile 9.0 compatible since the 21.0.0.2-beta release. Open Liberty is also the first vendor product to be added to the link:https://jakarta.ee/compatibility/#tab-9[Jakarta EE Platform 9.0 compatibility list], with the release of 21.0.0.3-beta.  That same beta release was then also used to add Open Liberty to the link:https://jakarta.ee/compatibility/#tab-9_1[Jakarta EE Platform 9.1 compatibility list].
```
@jhanders34

Reply:
That sounds good to me.